libraries/chain/controller.cpp

```diff
@@ -81,7 +81,7 @@ struct pending_state {
       if( std::holds_alternative<building_block>(_block_stage) )
          return std::get<building_block>(_block_stage)._pending_block_header_state;
 
-      return std::get<assembled_block>(_block_stage)._pending_block_header_state;     
+      return std::get<assembled_block>(_block_stage)._pending_block_header_state;
```

Comment:
Try to avoid unneeded diffs. It makes it more difficult to go back and see why a line was changed.

Reply:
Made whitespace changes.